slices/libtbb2.yaml

```diff
@@ -0,0 +1,19 @@
+package: libtbb2
+
+essential:
+  - libtbb2_copyright
+
+slices:
+  libs:
+    essential:
+      - libatomic1_libs
```

Comment:
As discussed in https://github.com/canonical/chisel-releases/pull/411/files#r1873020742, let's please comment this line out with appropriate notes since it's only a requirement for riscv64.

Reply:
done